### PR TITLE
CU-1tf7kpr - Picasso v0 runtime - part 1

### PR DESCRIPTION
**Pull request by hussein-aitlahcen**
This PR introduce simple pallets for the V0. 
I'll create a second, more specific one for Apollo.

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-1tf7kpr Picasso V0 runtime ugprade](https://app.clickup.com/t/1tf7kpr)

File: runtime/picasso/src/lib.rs [hunk 1]
```diff
@@ -673,6 +674,89 @@ impl democracy::Config for Runtime {
...
+	type Convert = sp_runtime::traits::ConvertInto;
+	type RelayChainAccountId = [u8; 32];
+	type InitialPayment = InitialPayment;
+	type VestingStep = VestingStep;
+	type Prefix = Prefix;
+	type WeightInfo = weights::crowdloan_rewards::WeightInfo<Runtime>;
+}
+
+parameter_types! {
+	  pub const MaxVestingSchedule: u32 = 2;
```
Comment (dzmitry-lahoda) [inline runtime/picasso/src/lib.rs:729]: very small?

**Review by dzmitry-lahoda — APPROVED** [review]

File: runtime/picasso/src/lib.rs [hunk 2]
```diff
@@ -745,6 +836,7 @@ pub type SignedExtra = (
 	system::CheckNonce<Runtime>,
 	system::CheckWeight<Runtime>,
 	transaction_payment::ChargeTransactionPayment<Runtime>,
+	crowdloan_rewards::PrevalidateAssociation<Runtime>,
```
Comment (seunlanlege) [inline runtime/picasso/src/lib.rs:839]: we should increase tx version for this

File: runtime/picasso/src/lib.rs (hunk 1 above)
Comment (hussein-aitlahcen) [reply runtime/picasso/src/lib.rs]: Updated!

File: runtime/picasso/src/lib.rs (hunk 2 above)
Comment (hussein-aitlahcen) [reply runtime/picasso/src/lib.rs:839]: Done!

**Review by dzmitry-lahoda — APPROVED** [review]

**Review by KaiserKarel — APPROVED** [review]

**Comment by KaiserKarel** [comment]
Discussed offline to merge this, to allow for collaborations.

File: runtime/picasso/src/lib.rs [hunk 3]
```diff
@@ -728,6 +813,12 @@ construct_runtime!(
 		DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 43,
 
 		Tokens: orml_tokens::{Pallet, Call, Storage, Event<T>} = 52,
+		Factory: currency_factory::{Pallet, Storage, Event<T>} = 53,
+		GovernanceRegistry: governance_registry::{Pallet, Call, Storage, Event<T>} = 54,
+		Assets: assets::{Pallet, Call, Storage} = 55,
+		CrowdloanRewards: crowdloan_rewards::{Pallet, Call, Storage, Event<T>} = 56,
+		Vesting: vesting::{Call, Event<T>, Pallet, Storage} = 57,
```
Comment (seunlanlege) [inline runtime/picasso/src/lib.rs:820]: @hussein-aitlahcen this pr doesn't add these pallets to the benchmark section

File: runtime/picasso/Cargo.toml [hunk 4]
```diff
@@ -72,6 +72,12 @@ common = { path = "../common", default-features = false }
 primitives = { path = "../primitives", default-features = false }
 composable-traits = { path = "../../frame/composable-traits", default-features = false }
 call-filter = { package = "pallet-call-filter", path = "../../frame/call-filter", default-features = false }
+currency-factory = { package = "pallet-currency-factory", path = "../../frame/currency-factory", default-features = false }
+governance-registry = { package = "pallet-governance-registry", path = "../../frame/governance-registry", default-features = false }
+assets = { package = "pallet-assets", path = '../../frame/assets', default-features = false }
+crowdloan-rewards = { package = "pallet-crowdloan-rewards", path = '../../frame/crowdloan-rewards', default-features = false }
+vesting = { package = "pallet-vesting", path = "../../frame/vesting", default-features = false }
+bonded-finance = { package = "pallet-bonded-finance", path = "../../frame/bonded-finance", default-features = false }
```
Comment (seunlanlege) [inline runtime/picasso/Cargo.toml:80]: missing std features as well as runtime-benchmark features

**Pull request merged**